content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -83,8 +83,10 @@ As an example, you can find detailed information about how `kube-up.sh` sets
 up logging for COS image on GCP in the corresponding
 [`configure-helper` script](https://github.com/kubernetes/kubernetes/blob/{{< param "githubbranch" >}}/cluster/gce/gci/configure-helper.sh).
 
-When using a **CRI container runtime**, the kubelet is responsible for rotating the logs and managing the logging directory structure. The kubelet
-sends this information to the CRI container runtime and the runtime writes the container logs to the given location. The two kubelet flags `container-log-max-size` and `container-log-max-files` can be used to configure the maximum size for each log file and the maximum number of files allowed for each container respectively.
+When using a **CRI container runtime**, the kubelet is responsible for rotating the logs and managing the logging directory structure.
+The kubelet sends this information to the CRI container runtime and the runtime writes the container logs to the given location.
+The two kubelet parameters `containerLogMaxSize` and `containerLogMaxFiles` in [kubelet config file](/docs/tasks/administer-cluster/kubelet-config-file/)
```

Comment:
I'd suggest we point the link to the config API page.

Reply:
I added the link to the config API page for the two parameters. But it'd better we also keep the link to the config file page so that user can know how to use these parameters. It's very useful for beginners to follow the example in the config file page to specify the parameters.